template.yml

```diff
@@ -84,6 +84,25 @@ Parameters:
     AllowedValues:
       - "true"
       - "false"
+  EnableAlerting:
+    Type: String
+    Description: >-
```

Comment:
Please mention alarms are for workload only

Reply:
Updated 👍